### ISSUE TITLE
Build error in mupdf when building the project

### ISSUE THREAD

**Issue by mdrokz** [Librum-Reader/Librum#40]
Hi im implementing docker support for building this project but im encountering build errors & im also planning to create a PR once im done.

i use ubuntu 22.04 image & QT 6.5.0 for building and im not sure which dependencies to install i tried the common ones but i still face these errors.

What OS do you use for building the project ?

here is the dockerfile https://github.com/mdrokz/Librum/blob/implement_docker/docker/linux/Dockerfile

here is the build error

```
cmake --build . -j $(nproc)
[  4%] Building mupdf (This takes a while) ...
[  4%] Built target librum_qmlimportscan
[  4%] Automatic MOC and UIC for target domain
[  4%] Built target domain_autogen
[ 14%] Built target domain
(+0.0s): parse.py:14:<module>: Warning, could not import clang: No module named 'clang'
(+0.0s): state.py:18:<module>: Warning: failed to import clang.cindex: e=ModuleNotFoundError("No module named 'clang'")
(+0.0s): state.py:18:<module>: We need Clang Python to build MuPDF python.
(+0.0s): state.py:18:<module>: Install with `pip install libclang` (typically inside a Python venv),
(+0.0s): state.py:18:<module>: or (OpenBSD only) `pkg_add py3-llvm.`
(+0.0s): --venv: __main__.py:2839:main2: running: "/usr/bin/python3" -m venv venv-mupdfwrap-3.10-x64
(+1.1s): --venv: __main__.py:2839:main2: [returned e=0]
(+1.1s): --venv: __main__.py:2857:main2: running: . venv-mupdfwrap-3.10-x64/bin/activate && python -m pip install --upgrade pip && python -m pip install --upgrade libclang swig && python ./scripts/mupdfwrap.py -d build/shared-release -b -j 0 m01 && deactivate
(+1.3s): --venv: __main__.py:2857:main2: Requirement already satisfied: pip in ./venv-mupdfwrap-3.10-x64/lib/python3.10/site-packages (23.2.1)
(+2.0s): --venv: __main__.py:2857:main2: Requirement already satisfied: libclang in ./venv-mupdfwrap-3.10-x64/lib/python3.10/site-packages (16.0.6)
(+2.2s): --venv: __main__.py:2857:main2: Requirement already satisfied: swig in ./venv-mupdfwrap-3.10-x64/lib/python3.10/site-packages (4.1.1)
(+2.4s): --venv: __main__.py:2857:main2: (+0.0s): -b: m: __main__.py:1535:build: Building libmupdf.so ...
(+2.4s): --venv: __main__.py:2857:main2: (+0.0s): -b: m: __main__.py:1185:_get_m_command: Setting -j to  multiprocessing.cpu_count()=16
(+2.4s): --venv: __main__.py:2857:main2: (+0.0s): -b: m: __main__.py:1537:build: running: cd /app/libs/mupdf && make -j 16 HAVE_GLUT=no HAVE_PTHREAD=yes verbose=yes shared=yes build=release build_prefix=shared-
(+2.4s): --venv: __main__.py:2857:main2: (+0.0s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: warning: -j16 forced in submake: resetting jobserver mode.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/jmemcust.c:19:10: fatal error: jinclude.h: No such file or directory
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    19 | #include "jinclude.h"
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |          ^~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: compilation terminated.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/jmemcust.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** Waiting for unfinished jobs....
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/color-lcms.c:38:10: fatal error: lcms2mt.h: No such file or directory
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    38 | #include "lcms2mt.h"
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |          ^~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: compilation terminated.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/color-lcms.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/font.c:31:10: fatal error: ft2build.h: No such file or directory
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    31 | #include <ft2build.h>
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |          ^~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: compilation terminated.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpx.c:63:10: fatal error: openjpeg.h: No such file or directory
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    63 | #include <openjpeg.h>
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |          ^~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: compilation terminated.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/load-jpx.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/font.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/harfbuzz.c:30:10: fatal error: hb.h: No such file or directory
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    30 | #include "hb.h"
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |          ^~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: compilation terminated.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: In file included from source/fitz/output-jpeg.c:25:
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: /usr/include/jpeglib.h:69:9: error: unknown type name ‘JSAMPLE’
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    69 | typedef JSAMPLE *JSAMPROW;      /* ptr to one image row of pixel samples. */
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |         ^~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: /usr/include/jpeglib.h:658:3: error: unknown type name ‘JSAMPLE’
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   658 |   JSAMPLE *sample_range_limit;  /* table for fast range-limiting */
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |   ^~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/harfbuzz.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/output-docx.c:28:10: fatal error: extract/extract.h: No such file or directory
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    28 | #include "extract/extract.h"
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |          ^~~~~~~~~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: compilation terminated.
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/output-docx.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: In file included from source/fitz/load-jpeg.c:30:
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: /usr/include/jpeglib.h:69:9: error: unknown type name ‘JSAMPLE’
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:    69 | typedef JSAMPLE *JSAMPROW;      /* ptr to one image row of pixel samples. */
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |         ^~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/output-jpeg.c: In function ‘fz_write_pixmap_as_jpeg’:
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/output-jpeg.c:173:17: error: ‘JPEG_LIB_VERSION’ undeclared (first use in this function)
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   173 |                 jpeg_create_compress(&cinfo);
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                 ^~~~~~~~~~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/output-jpeg.c:173:17: note: each undeclared identifier is reported only once for each function it appears in
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: /usr/include/jpeglib.h:658:3: error: unknown type name ‘JSAMPLE’
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   658 |   JSAMPLE *sample_range_limit;  /* table for fast range-limiting */
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |   ^~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/output-jpeg.c:217:40: warning: assignment to ‘JSAMPROW’ {aka ‘int *’} from incompatible pointer type ‘unsigned char *’ [-Wincompatible-pointer-types]
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   217 |                         row_pointer[0] = &pix->samples[cinfo.next_scanline * pix->stride];
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                                        ^
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/output-jpeg.o] Error 1
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpeg.c: In function ‘fz_load_jpeg’:
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpeg.c:390:17: error: ‘JPEG_LIB_VERSION’ undeclared (first use in this function)
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   390 |                 jpeg_create_decompress(&cinfo);
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                 ^~~~~~~~~~~~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpeg.c:390:17: note: each undeclared identifier is reported only once for each function it appears in
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpeg.c:446:53: warning: passing argument 2 of ‘jpeg_read_scanlines’ from incompatible pointer type [-Wincompatible-pointer-types]
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   446 |                         jpeg_read_scanlines(&cinfo, row, 1);
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                                                     ^~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                                                     |
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                                                     unsigned char **
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: In file included from source/fitz/load-jpeg.c:30:
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: /usr/include/jpeglib.h:999:51: note: expected ‘JSAMPARRAY’ {aka ‘int **’} but argument is of type ‘unsigned char **’
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   999 |                                        JSAMPARRAY scanlines,
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                                        ~~~~~~~~~~~^~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpeg.c: In function ‘fz_load_jpeg_info’:
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: source/fitz/load-jpeg.c:513:17: error: ‘JPEG_LIB_VERSION’ undeclared (first use in this function)
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:   513 |                 jpeg_create_decompress(&cinfo);
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build:       |                 ^~~~~~~~~~~~~~~~~~~~~~
(+2.6s): --venv: __main__.py:2857:main2: (+0.2s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: make[3]: *** [Makefile:126: build/shared-release/source/fitz/load-jpeg.o] Error 1
(+2.8s): --venv: __main__.py:2857:main2: (+0.4s): -b: m: __main__.py:1537:build: (+0.0s): -b: m: __main__.py:1537:build: 
(+2.8s): --venv: __main__.py:2857:main2: (+0.4s): -b: m: __main__.py:1537:build: [returned e=2]
(+2.8s): --venv: __main__.py:2857:main2: Traceback (most recent call last):
(+2.8s): --venv: __main__.py:2857:main2:     scripts/mupdfwrap.py:6:<module>(): wrap.__main__.main()
(+2.8s): --venv: __main__.py:2857:main2:     scripts/wrap/__main__.py:2907:main(): jlib.exception_info()
(+2.8s): --venv: __main__.py:2857:main2:     ^except raise:
(+2.8s): --venv: __main__.py:2857:main2:     scripts/wrap/__main__.py:2905:main(): main2()
(+2.8s): --venv: __main__.py:2857:main2:     scripts/wrap/__main__.py:2295:main2(): build( build_dirs, swig_command, args, vs_upgrade)
(+2.8s): --venv: __main__.py:2857:main2:     scripts/wrap/__main__.py:1537:build(): jlib.system( command, prefix=jlib.log_text(), out='log', verbose=1)
(+2.8s): --venv: __main__.py:2857:main2:     scripts/jlib.py:1682:system(): raise Exception( message)
(+2.8s): --venv: __main__.py:2857:main2: Exception: Command failed: cd /app/libs/mupdf && make -j 16 HAVE_GLUT=no HAVE_PTHREAD=yes verbose=yes shared=yes build=release build_prefix=shared-
(+2.8s): --venv: __main__.py:2857:main2: [returned e=1]
Traceback (most recent call last):
    scripts/mupdfwrap.py:6:<module>(): wrap.__main__.main()
    scripts/wrap/__main__.py:2907:main(): jlib.exception_info()
    ^except raise:
    scripts/wrap/__main__.py:2905:main(): main2()
    scripts/wrap/__main__.py:2857:main2(): jlib.system(command, out='log', verbose=1)
    scripts/jlib.py:1682:system(): raise Exception( message)
Exception: Command failed: . venv-mupdfwrap-3.10-x64/bin/activate && python -m pip install --upgrade pip && python -m pip install --upgrade libclang swig && python ./scripts/mupdfwrap.py -d build/shared-release -b -j 0 m01 && deactivate
gmake[2]: *** [src/application/CMakeFiles/mupdf.dir/build.make:73: ../libs/mupdf/build/shared-release/libmupdfcpp.so] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:353: src/application/CMakeFiles/mupdf.dir/all] Error 2
gmake: *** [Makefile:136: all] Error 2
```

**Comment by DavidLazarescu** [comment]
Hi, why exactly are you trying to add docker support to the application? To me it would make more sense to run the server in a docker.

**Comment by fristhon** [comment]
Try to run that line manually by following this steps:
first of all be sure you are using the correct python environment:
`./venv-mupdfwrap-3.10-x64/bin/activate`

Then change your path to scripts directory and execute:
`mupdfwrap.py -d build/shared-release -b -j 0 m01`

By this, you will get a more specific error.

**Comment by mdrokz** [comment]
> Hi, why exactly are you trying to add docker support to the application? To me it would make more sense to run the server in a docker, but I don't see why I'd run the client in a docker.

Hey sorry for the late reply, yes docker is mostly used to containerize applications. 

But another use case for containerizing client side applications is if i want to easily build the application & test a change installing all of the build dependencies is time consuming & hard sometimes. docker maintains the same environment so you can easily build & run the application.

Hope this makes sense!

**Comment by mdrokz** [comment]
> Try to run that line manually by following this steps: first of all be sure you are using the correct python environment: `./venv-mupdfwrap-3.10-x64/bin/activate`
> 
> Then change your path to scripts directory and execute: `mupdfwrap.py -d build/shared-release -b -j 0 m01`
> 
> By this, you will get a more specific error.

Sure let me try this when i get some free time

**Comment by DavidLazarescu** [comment]
@mdrokz the dependencies you listed look all right to me but there seems to be an issue with some jpeg libraries from the errors I see. Does that machine have `libjpeg` installed?

**Comment by mdrokz** [comment]
> @mdrokz the dependencies you listed look all right to me but there seems to be an issue with some jpeg libraries from the errors I see. Does that machine have `libjpeg` installed?

Yes i installed libjpeg manually in the container but it still gave the same error

**Comment by DavidLazarescu** [comment]
Do you also have `libjpeg-dev`?

**Comment by mdrokz** [comment]
> Do you also have `libjpeg-dev`?

yes i installed that as well

**Comment by DavidLazarescu** [comment]
This problem might be fixed with the release of Qt 6.6, please try downloading it via the Qt Maintenance tool and changing 6.5 to 6.6 here: https://github.com/Librum-Reader/Librum/blob/main/CMakeLists.txt#L25

I will upgrade Librum to 6.6 in the following days as well.
